infra/Pos/Slotting/Ntp.hs

```diff
@@ -65,12 +70,21 @@ makeLenses ''NtpSlottingState
 ----------------------------------------------------------------------------
 
 -- | Monad transformer which implements NTP-based solution for slotting.
-type NtpSlotting = Ether.ReaderT NtpSlottingVar
+-- Flag means whether to use real NTP servers or rely on local time.
+type NtpSlotting = Ether.ReaderT (Bool, NtpSlottingVar)
```

Comment:
@int-index regarding `local'`, I have written it, because it's absent in the `ether`.... (`withReaderT`). it's so unexpected :(